packages/kokkos-kernels/src/Kokkos_ArithTraits.hpp

```diff
@@ -57,7 +57,6 @@
 #include <cfloat>
 #include <climits>
 #include <cmath>
-#include <cstdlib> // strtof, strtod, strtold
```

Comment:
I approve in general of removing includes.  However, please be sure to test all downstream packages, in case they implicitly depend on this include.